cds-plugin.js

```diff
@@ -47,13 +47,16 @@ const rmFiles = async (dir, exts) => fs.existsSync(dir)
     )
     : undefined
 
+// FIXME: remove once cds7 has been phased out
+const isAtLeastCds8 = cds?.version && cds.version >= '8.0.0'
```

Comment:
Why not using:
`if ( cds?.version && cds.version < '8.0.0') return`

Note: in general using `>` `<` operators for semver versions comparing might produce wrong results, but here it should be ok ;-)

Reply:
isn't that just the De Morgan'd version of what I used? Or is there a difference/ advantage I am overlooking? :D

Reply:
- Yes, I used the negotiation to return immediately, you can of course also use your version :) 
My point was - why not check once and return immediately without registering the build plugin which won't work anyway?